### ISSUE TITLE
Windows 11 VBS (Virtualization-based Security) appears Not enabled on System Information

### ISSUE THREAD

**Issue by zirblazer** [Dasharo/dasharo-issues#539]
### Device

MSI PRO Z690-A WIFI DDR4

### Dasharo version

1.1.2

### Affected component(s) or functionality

VBS (Virtualization-based Security)

### Brief summary

x

### How to reproduce

Both me and miczyg have VBS (Virtualization-based Security) as Not enabled on Windows 11. This can be checked by opening System Information and looking for Virtualization-based Security on System Summary.
One of VBS prerequisites is to enable Memory integrity, which can be found at Windows Security / Device security / Core isolation menu. miczyg reported an INACCESSIBLE_BOOT_DEVICE BSOD after enabling it.

Note that I'm positively aware that it MAY have been at some point functional because I recall having disabled some Windows security features because Intel XTU (Extreme Tuning Utility) requires VBS to be disabled and both me and miczyg toyed with that. But I don't recall if this was with MSI Firmware or with Dasharo, so it means that it may have been broken all along but no one noticed it.
Also note that this means than certain BSODs when migrating from MSI Firmware to Dasharo could actually be caused by Memory integrity being enabled in Windows but failing on Dasharo. I have no idea about Windows requeriments to allow you to enable it if it isn't actually functional. I have seen some people suggesting to disable Intel VT-x on Firmware to solve the BSODs issues if you lock yourself out of Windows, but Dasharo doesn't expose VT-x control (It is always enabled) so you can't workaround it this way.

Basically, more investigation needed.
Basic VBS info: https://learn.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-vbs

### Expected behavior

x

### Actual behavior

x

**Comment by pietrushnic** [comment]
@miczyg1 insights from twitter
> Ohh my! I have checked the requirements on MS Documentation and we may lack WSMT table and MOR v2. In case of MAT I would have to check. But thanks for pointing it. Will put it to the backlog.

And from Dwizzzle:

> Yeah it won't automatically enabled without the WSMT table (this is a controversial decision in my opinion :P) but users can still auto-enable.   Feature request: Integrate FASR or PPAM to enable Windows Secure Launch and upcharge for it!  The 14900k with a full opensource SCPC!

**Comment by pietrushnic** [comment]
Fact how it is validated may add some knowledge about implementation and compliance expectations:
- https://bugzilla.tianocore.org/show_bug.cgi?id=4374
- https://bugzilla.tianocore.org/show_bug.cgi?id=4419

**Comment by mateusz1987** [comment]
I want to report that the issue also occurs on the MSI PRO Z790-P WIFI DDR5 motherboard with Dasharo version 0.91. The problem is significant because enabling VBA greatly enhances the security of Windows 11 through core isolation. It would also allow the use of Windows Sandbox or additional Hyper-V virtual machines. Furthermore, in a year, Windows 10 will lose its security update support

**Comment by Hyperstructx** [comment]
@zirblazer @pietrushnic 👍 

This is extremely important.

**Comment by pietrushnic** [comment]
@Hyperstructx agree.

@BeataZdunczyk, it was not released in the milestone you added a week ago, so I guess it would have to be moved to the next one?

**Comment by FlexApex** [comment]
I had this exact problem on a NovaCustom NV54 notebook (Intel 14th gen) with Dasharo 1.0.0. Windows wouldn't run hyper-V VMs or WSL v2 guests. After upgrading to 1.0.1 it works!
